commit 1d74626eb39ac238bbf5c5c19429e5ac204ff23e
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-03-26T22:47:56+00:00 (7 months ago)
    
    OS-7670 kvm: Remove unused thermal MSRs
    Reviewed by: John Levon <john.levon@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/msr-index.h b/msr-index.h
index bea940f..f059650 100644
--- a/msr-index.h
+++ b/msr-index.h
@@ -240,20 +240,6 @@
 #define MSR_IA32_MPERF			0x000000e7
 #define MSR_IA32_APERF			0x000000e8
 
-#define MSR_IA32_THERM_CONTROL		0x0000019a
-#define MSR_IA32_THERM_INTERRUPT	0x0000019b
-
-#define THERM_INT_LOW_ENABLE		(1 << 0)
-#define THERM_INT_HIGH_ENABLE		(1 << 1)
-
-#define MSR_IA32_THERM_STATUS		0x0000019c
-
-#define THERM_STATUS_PROCHOT		(1 << 0)
-
-#define MSR_THERM2_CTL			0x0000019d
-
-#define MSR_THERM2_CTL_TM_SELECT	(1ULL << 16)
-
 #define MSR_IA32_MISC_ENABLE		0x000001a0
 
 /* MISC_ENABLE bits: architectural */
